### ISSUE TITLE
Iframes lack accessible names

### ISSUE THREAD

**Issue by BuddyinKC** [omeka-s-themes/freedom#61]
Our DubBot auditing tool is flagging some iframes that lack accessible names. Here are four examples:

- https://exhibits.library.umkc.edu/s/mcilrath-swhof/item/4674
- https://exhibits.library.umkc.edu/s/mcilrath-swhof/media/4707
- https://exhibits.library.umkc.edu/s/antiguedades-mexicanas/item/4860
- https://exhibits.library.umkc.edu/s/antiguedades-mexicanas/item/4858

**Comment by aviannamiller** [comment]
A fix for this was included in the Omeka S v4.2.0 release. The iframe title pulls from configured alt text on media.
